src/platform/packages/shared/kbn-esql-composer/src/commands/from.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { parse } from '@kbn/esql-ast';
+import { createPipeline } from '../create_pipeline';
+import type { QueryPipeline } from '../types';
+import { escapeColumn } from '../utils/formatters';
+
+export function from(...patterns: Array<string | string[]>): QueryPipeline {
+  const allPatterns = patterns.flatMap((pattern) => pattern);
+
+  const { root } = parse(`FROM ${allPatterns.map((pattern) => escapeColumn(pattern)).join(',')}`);
```

Comment:
I am not sure that escaping each pattern is valid in ES|QL. Have you tested it?

Reply:
True! should the ast escape the pattern (I don't know if it already does it)